locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -1365,6 +1292,18 @@ msgstr ""
 "Weitere Informationen finden Sie in der Datenschutzerklärung von Vimeo "
 "unter: "
 
+#~ msgid "Blocked by moderator on"
+#~ msgstr "Von Moderation gesperrt"
+
+#~ msgid "Submit"
+#~ msgstr "Einreichen"
+
+#~ msgid "Other"
+#~ msgstr "Sonstige"
+
+#~ msgid "New answer"
+#~ msgstr "Neue Antwort"
+
```

Comment:
These aren't that many translations lost. And they all seem to be from a4. Did you check if we still have the strings there?

Reply:
so it seems that they are still in use, some not in JSX and some of them are indeed used in JSX.
I checked with latest a4 main and the a4 commit which is used in current mB main?

curiosity: how do you know that they are from a4 @fuzzylogic2000 ?

Reply:
I noticed that a lot of the lines that were removed were the lines saying that the strings were from a4. I mean the ones above the strings.